src/http/response.ts

```diff
@@ -114,7 +150,10 @@ export class Response {
    * @return string|boolean
    *     The html content of the view, or false if the `views_path` is not set.
    */
-  public render(...args: any): string | boolean {
+  public render(
+    // deno-lint-ignore no-explicit-any
+    ...args: any
```

Comment:
`...args: unknown` just to match the doc block?

Reply:
addressed: ba8108d